CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Add `classes` and `rowClasses` props to `ProductDescription` component. 
```

Comment:
It's better to say also that you removed some unused files.

Reply:
I didn't, those commits came from rebase